### ISSUE TITLE
Where can i see the full list of kwargs for the scene.show method?

### ISSUE THREAD

**Issue by ManuGraiph** [mikedh/trimesh#1649]
As the title says, i can't find the list of arguments anywhere to fully customize how the points/mesh look when using pyglet to plot a scene with scene.show, i've only seen a couple here in some github posts, but other than that, nothing.

**Comment by mikedh** [comment]
Hey, the keyword arguments to `scene.show` are passed through to `trimesh.viewer.windowed.SceneViewer` constructor:
https://trimsh.org/trimesh.viewer.windowed.html#trimesh.viewer.windowed.SceneViewer.__init__

**Comment by ManuGraiph** [comment]
Yes, i know, but, for example.. what args can i pass with ` line_settings`? I know one of its keys is `point_size`, but what other i can use for i don't know.. point symbol, arist color, etc. when using `view()`?
